core/templates/dev/head/services/searchService.js

```diff
@@ -64,18 +64,39 @@ oppia.factory('searchService', [
     return _searchCursor === null;
   };
 
+  var updateSearchFields = function(itemsType, urlComponent,
+                                    selectionDetails) {
+    var itemCodeGroup = urlComponent.match(/\([A-Za-z%20" ]+\)/);
+
+    if (itemCodeGroup != '') {
+      var itemCodes = itemCodeGroup[0].replace('("', '');
```

Comment:
Nope -- check that the first two characters are (", check that the last two are "), then take the substring of the original string that omits the first two and last two characters. This is stricter than replacing occurrences of (" anywhere, and we want to be strict.